src/gui/topbar_subwindows/gui_diplomacy_window.hpp

```diff
@@ -1083,6 +1083,43 @@ class great_power_inf_detail : public window_element_base {
 	}
 };
 
+template<class T>
+class nation_status_diplomacy_window : public simple_text_element_base {
```

Comment:
This duplicates the contents of an element in gui_common_elements 585. If we need to use it more than once, the function status enum -> text should probably be factored out (in case it changes, so that one of the two ui elements won't end up out of sync)